### ISSUE TITLE
what's the "labels" here in RNNsearch.py ?

### ISSUE THREAD

**Issue by cyzhangAThit** [thumt/THUMT#19]
Hi, I try to run the thumt with the RNNsearch model, but I found couldn't.
A variable named "labels" seems  not be defined.
==========Line 307===============
    ce = layers.nn.smoothed_softmax_cross_entropy_with_logits(
        logits=logits,
        labels=labels,
        smoothing=params.label_smoothing,
        normalize=True
    )

**Comment by Playinf** [comment]
Thank you for reporting, the bug is now fixed. `labels` here is equivalent to `features["target"]`.
